src/main.c

```diff
@@ -192,6 +190,60 @@ void kill_childs()
 	debug(D_EXIT, "All threads/childs stopped.");
 }
 
+struct option_def options[] = {
+	// opt description                                                       arg name                     default value
+	{'c', "Load alternate configuration file",                               "config_file",                          CONFIG_DIR "/" CONFIG_FILENAME},
+	{'D', "Disable fork into background",                                    NULL,                                   NULL},
+	{'h', "Display help message",                                            NULL,                                   NULL},
+	{'P', "File to save a pid while running",                                "FILE",                                 NULL},
+	{'p', "Port to listen. Can be from 1 to 65535.",                         "port_number",                          "19999"},
+	{'s', "Path to access host /proc and /sys when running in a container.", "PATH",                                 NULL},
+	{'t', "The frequency in seconds, for data collection. \
+Same as 'update every' config file option.",                                 "seconds",                              "1"},
+	{'l', "The number of entries the netdata daemon will keep in memory \
+for each chart dimension. Same as 'history config file option.",             "lines_to_save",                        NULL},
```

Comment:
Yes, this is history. Do you want to change the description or the option flag?
Why I did not change the option flag:
1. I tried not to break configuration options which fit to the standard.
2. `-h` should be help message.
3. `-H`: Capitals are intended to be negative options like `-D`. I know I break this with `-P`.